test/interactions/dragInteractionTests.ts

```diff
@@ -1,376 +1,410 @@
+        TestMethods.triggerFakeMouseEvent("mouseup", target, endPoint.x, endPoint.y, 2);
+        assert.isTrue(endCallbackCalled, "callback was not called on mouseup from the right-click button");
+        TestMethods.triggerFakeMouseEvent("mouseup", target, endPoint.x, endPoint.y); // end the drag
+
+        receivedStart = null;
+        receivedEnd = null;
+        TestMethods.triggerFakeTouchEvent("touchstart", target, [{x: startPoint.x, y: startPoint.y}]);
+        TestMethods.triggerFakeTouchEvent("touchend", target, [{x: endPoint.x, y: endPoint.y}]);
+        assert.isTrue(endCallbackCalled, "callback was called on drag ending (touchend)");
+        assert.deepEqual(receivedStart, startPoint, "was passed the correct starting point");
+        assert.deepEqual(receivedEnd, endPoint, "was passed the correct current point");
+
+        dragInteraction.offDragEnd(endCallback);
+
+        endCallbackCalled = false;
+        TestMethods.triggerFakeMouseEvent("mousedown", target, startPoint.x, startPoint.y);
+        TestMethods.triggerFakeMouseEvent("mouseup", target, endPoint.x, endPoint.y);
+        assert.isFalse(endCallbackCalled, "callback was called on drag ending (mouseup)");
+
+        TestMethods.triggerFakeTouchEvent("touchstart", target, [{ x: startPoint.x, y: startPoint.y }]);
+        TestMethods.triggerFakeTouchEvent("touchend", target, [{ x: endPoint.x, y: endPoint.y }]);
+        assert.isFalse(endCallbackCalled, "callback decoupled from interaction");
+
+        svg.remove();
+      });
+
+      it("can register two onDragEnd() callbacks on the same Component", () => {
+        let endCallback1Called = false;
+        let endCallback2Called = false;
+        let endCallback1 = () => endCallback1Called = true;
+        let endCallback2 = () => endCallback2Called = true;
+
+        dragInteraction.onDragEnd(endCallback1);
+        dragInteraction.onDragEnd(endCallback2);
+
+        dragInteraction.attachTo(component);
+
+        let target = component.background();
+
+        TestMethods.triggerFakeMouseEvent("mousedown", target, startPoint.x, startPoint.y);
+        TestMethods.triggerFakeMouseEvent("mouseup", target, endPoint.x, endPoint.y);
+        assert.isTrue(endCallback1Called, "callback 1 was called on drag ending (mouseup)");
+        assert.isTrue(endCallback2Called, "callback 2 was called on drag ending (mouseup)");
+
+        endCallback1Called = false;
+        endCallback2Called = false;
+        dragInteraction.offDragEnd(endCallback1);
+        TestMethods.triggerFakeMouseEvent("mousedown", target, startPoint.x, startPoint.y);
+        TestMethods.triggerFakeMouseEvent("mouseup", target, endPoint.x, endPoint.y);
+        assert.isFalse(endCallback1Called, "callback 1 was disconnected from the drag end interaction");
+        assert.isTrue(endCallback2Called, "callback 2 is still connected to the drag end interaction");
+
+        svg.remove();
+      });
+
+      it("can register multiple callbacks", () => {
```

Comment:
It's not clear how this differs from the previous tests for multiple callbacks. Can we clarify the name or remove the test if it's extraneous?